src/test/java/org/gridsuite/cgmes/gl/server/utils/TestUtils.java

```diff
@@ -0,0 +1,47 @@
+package org.gridsuite.cgmes.gl.server.utils;
+
+import com.powsybl.iidm.network.Country;
+import com.powsybl.iidm.network.Line;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.Substation;
+import com.powsybl.iidm.network.extensions.LinePosition;
+import com.powsybl.iidm.network.extensions.SubstationPosition;
+import org.gridsuite.cgmes.gl.server.dto.LineGeoData;
+import org.gridsuite.cgmes.gl.server.dto.SubstationGeoData;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public final class TestUtils {
```

Comment:
author

Reply:
why? there the git history and git blame, and I'm not really the author as I just moved the code...